src/NServiceBus.AcceptanceTests/Routing/AutomaticSubscriptions/When_starting_an_endpoint_with_a_saga.cs

```diff
@@ -38,7 +38,7 @@ class Subscriber : EndpointConfigurationBuilder
             public Subscriber()
             {
                 EndpointSetup<DefaultServer>(c => c.Pipeline.Register("SubscriptionSpy", typeof(SubscriptionSpy), "Spies on subscriptions made"))
-                    .AddMapping<MyEventWithParent>(typeof(Subscriber)) //just map to our self for this test
+                    .AddMapping<MyEventBase>(typeof(Subscriber)) //just map to our self for this test
```

Comment:
this was a test relying on the route inheritance (having only a handler for a base class but a route for the subclass). According to the new behavior, there has to be a mapping for the type of event you want to receive.